apps/api/src/hyp3_api/routes.py

```diff
@@ -179,5 +181,6 @@ def patch(self, subscription_id):
 app.add_url_rule('/user', view_func=user_view)
 
 subscriptions_view = Subscriptions.as_view('subscriptions', api_spec)
-app.add_url_rule('/subscriptions/<subscription_id>', view_func=subscriptions_view, methods=['PATCH'])
-app.add_url_rule('/subscriptions', view_func=subscriptions_view, methods=['GET', 'POST'])
+app.add_url_rule('/subscriptions/<subscription_id>', view_func=subscriptions_view, methods=['PATCH', 'GET'])
+app.add_url_rule('/subscriptions', view_func=subscriptions_view, defaults={'subscription_id': None})
```

Comment:
if we're mimicking the `/jobs` setup, should specify GET here
```suggestion
app.add_url_rule('/subscriptions', view_func=subscriptions_view, methods=['GET'], defaults={'subscription_id': None})
```